metadata_backend/api/health.py

```diff
@@ -30,9 +33,31 @@ async def get_health_status(self, _: Request) -> Response:
             services["database"] = {"status": "Ok"} if conn < 1000 else {"status": "Degraded"}
         else:
             services["database"] = {"status": "Down"}
+
+        # Determine the status of loaded services
+
+        metax_handler = MetaxServiceHandler()
+        datacite_handler = DataciteServiceHandler()
+        rems_handler = RemsServiceHandler()
```

Comment:
I have tried to keep only one instance of the service handlers, passing them to the relevant classes in `__init__`. Perhaps it would be nice to keep with the consistency?